compile.go

```diff
@@ -429,9 +425,6 @@ func (t *TableObject) PolyType() semantic.PolyType {
 func (t *TableObject) Str() string {
 	panic(values.UnexpectedKind(semantic.Object, semantic.String))
 }
-func (t *TableObject) Bytes() []byte {
```

Comment:
What are these changes?  Seems unrelated to the KAMA work?

Reply:
i think i broke it when rebasing. fixing rn